src/core_modules/capture-core-utils/rulesEngine/rulesEngine.types.js

```diff
@@ -57,15 +57,16 @@ export type ProgramRuleAction = {
     programStageId?: ?string,
     programStageSectionId?: ?string,
     trackedEntityAttributeId?: ?string,
+    programRuleVariableId?: ?string,
     optionGroupId: ?string,
     optionId: ?string,
     style?: ?Object,
 };
 
 export type ProgramRule = {
     id: string,
-    name: string;
```

Comment:
I wonder why didn't the linter detect these typos. Do you have any idea @superskip?

Reply:
It must be lacking a detection rule for that sort of thing, can't imagine any other explanation.